File: Dev/Frontend/LambdaPP_WebApp_Alpha_3.0/WildTask/src/app/logic/models/Low_Stock.ts
export default class Low_Stock {

    stock_ID!: number
    week: Boolean = false
    bi_Week: Boolean = false
    month: Boolean = false;
    stock_Quantity!: number;
    usage_Weekly!: number;
    usage_Bi_Weekly!: number;
    usage_Monthly!: number;

    constructor(quantity: number)
    {
        this.stock_Quantity = quantity
    }
}

File: Dev/WebApp/LambdaPP_ProjectCore_Alpha/LamdbdaPP_WebAPI_Alpha/Database/Models/Low_Stock.cs
using LamdbdaPP_WebAPI_D3.Database.Models;
using System;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace LambdaPP_WebAPI_Alpha.Database.Models
{
    public class Low_Stock
    {

        [JsonPropertyName("stock_ID")]
        public int Stock_ID { get; set; }

        [JsonPropertyName("week")]
        public bool Week { get; set; }

        [JsonPropertyName("bi_Week")]
        public bool Bi_Week { get; set; }

        [JsonPropertyName("month")]
        public bool Month { get; set; }

        [JsonPropertyName("stock_Quantity")]
        public int Stock_Quantity { get; set; }

        [JsonPropertyName("usage_Weekly")]
        public int Usage_Weekly { get; set; }

        [JsonPropertyName("usage_Bi_Weekly")]
        public int Usage_Bi_Weekly { get; set; }

        [JsonPropertyName("usage_Monthly")]
        public int Usage_Monthly { get; set; }


    }
}
